src/components/OSCALSystemImplementation.js

```diff
@@ -44,7 +45,9 @@ export default function OSCALSystemImplementation(props) {
               <OSCALSectionHeader>System Implementation</OSCALSectionHeader>
             </Grid>
             <Grid item xs={12}>
-              <Typography>{props.systemImplementation.remarks}</Typography>
+              <Typography>
+                <OSCALMarkdownProse text={props.systemImplementation.remarks} />
+              </Typography>
```

Comment:
Is this `Typography` even needed?

Reply:
Without the `Typography`, the font of the system implementation remarks changes so it is not consistent with the rest of the page. The first screenshot shows it with the `Typography` and the second screenshot shows it without the `Typography`:
![Screenshot (18)](https://user-images.githubusercontent.com/81392411/183676120-a98d4c95-a0a5-4bae-86e7-c3e38e792cf2.png)
![Screenshot (16)](https://user-images.githubusercontent.com/81392411/183676136-6ee916d6-3c81-4970-92aa-10531811a9da.png)